### PR TITLE
feat: add font weight prop

### PR DESCRIPTION
**Pull request by MatthewKyle22**
Adds fontWeight prop to ActionBarItem component which passes to UIText for style control

### CODE REVIEW

**Review by redreceipt — COMMENTED** [review]
I'm good with it, wanna write me a story and test?

File: packages/apollos-ui-kit/src/ActionBar/ActionBarItem.js [hunk 1]
```diff
@@ -29,7 +30,7 @@ const ActionBarItem = ({
       <Icon fill={tint} size={size} name={icon} />
 
       {/* using stlye object here is just as efficient as creating a new styled component with prop, since UIText is already a styled component */}
-      <UIText style={{ color: tint }}>{label}</UIText>
+      <UIText style={{ color: tint, fontWeight }}>{label}</UIText>
```
Comment (conrad-vanl) [inline packages/apollos-ui-kit/src/ActionBar/ActionBarItem.js:33]: I honestly think the better solve here is to allow a `style` prop to be passed down

Comment (redreceipt) [reply packages/apollos-ui-kit/src/ActionBar/ActionBarItem.js:33]: or just "name" that `UIText` component so they can override the style prop? that matches better and doesn't extend the API

**Comment by conrad-vanl** [comment]
Closing in favor of other solutions discussed above. We can re-open when changes are made.

**Pull request closed without merging**